packages/0x.js/webpack.config.js

```diff
@@ -39,6 +39,9 @@ module.exports = {
             }),
         ],
     },
+    node: {
+        fs: 'empty',
+    },
```

Comment:
I'm still worried about this hack. Does this mean anyone using `0x.js` in a webpack project will need to do this too to get it to work? If so, please add browser-specific instructions to the README explaining the need for this step.

Reply:
I created a test project importing 0x.js and tried bundling with webpack. Sadly yes and there doesn't seem to be any workaround  as long as we are using `ethereumjs-vm` as a dependency. I'll add instructions to the README.